nelmio/api-doc-bundle/2.13/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "bundles": {
+        "Nelmio\\ApiDocBundle\\NelmioApiDocBundle": ["all"]
```

Comment:
I am fifty/fifty on this issue, an API can be public and documentation open.
Also in the [official documentation ](https://symfony.com/doc/current/bundles/NelmioApiDocBundle/index.html)of the bundle is declared for production.